pkg/plugins/discovery.go

```diff
@@ -81,7 +106,7 @@ func (l *localRegistry) Plugin(name string) (*Plugin, error) {
 			return readPluginInfo(name, p)
 		}
 	}
-	return nil, ErrNotFound
+	return nil, errors.Wrapf(ErrNotFound, "could not find plugin %s in v1 plugin registry", name)
```

Comment:
can you add the plugin name to the pluginv2 error msg as well? https://github.com/moby/moby/blob/6d4d3c52ae7c3f910bfc7552a2a673a8338e5b9f/plugin/backend_linux.go#L527